jwst/emicorr/emicorr.py

```diff
@@ -454,6 +454,9 @@ def apply_emicorr(input_model, emicorr_model,
         for k in range(4):
             noise[:, :, :, noise_x + k] = dd_noise
 
+        # Safety catch; anywhere the noise value is not finite, set it to zero
+        noise[np.isfinite(noise) == False] = 0
```

Comment:
One of the checks is complaining that the syntax is not safe, I recommend to change it to noise[~np.isfinite(noise)] = 0